packages/@aws-cdk/aws-codebuild/lib/project.ts

```diff
@@ -1157,9 +1182,12 @@ export class LinuxBuildImage implements IBuildImage {
   public static readonly STANDARD_1_0 = LinuxBuildImage.codeBuildImage('aws/codebuild/standard:1.0');
   public static readonly STANDARD_2_0 = LinuxBuildImage.codeBuildImage('aws/codebuild/standard:2.0');
   public static readonly STANDARD_3_0 = LinuxBuildImage.codeBuildImage('aws/codebuild/standard:3.0');
+
   public static readonly AMAZON_LINUX_2 = LinuxBuildImage.codeBuildImage('aws/codebuild/amazonlinux2-x86_64-standard:1.0');
   public static readonly AMAZON_LINUX_2_2 = LinuxBuildImage.codeBuildImage('aws/codebuild/amazonlinux2-x86_64-standard:2.0');
-  public static readonly AMAZON_LINUX_2_ARM = LinuxBuildImage.codeBuildImage('aws/codebuild/amazonlinux2-aarch64-standard:1.0');
+
```

Comment:
Why is the usage of LinuxBuildImage and ArmBuildImage different? One uses a static method and one uses a constructor... lets normalize.

Reply:
It's because we don't want to expose the fact that `LinuxBuildImage` is the implementation used for methods like `LinuxBuildImage.codeBuildImage()`, `LinuxBuildImage. fromDockerRegistry()`, etc.  - that's why the constructor of `LinuxBuildImage` is private. We don't need to do the same for `ArmBuildImage`, as that entire class is module-private already.

Reply:
Sounds good